src/component/GiftCategory.styles.ts

```diff
@@ -0,0 +1,36 @@
+import styled from '@emotion/styled'
+import { theme } from '@/theme'
+
+export const Container = styled.div`
+  margin: ${theme.spacing[4]} ${theme.spacing[0]};
+  padding: ${theme.spacing[4]};
+  background-color: ${theme.colors.colorScale.gray[0]};
+`
+
+export const Title = styled.h2`
+  ${theme.typography.title1Bold};
+`
+
+export const Grid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(5, minmax(0, 1fr));
+  gap: ${theme.spacing[3]};
```

Comment:
styled를 사용한다면 props 통해서 theme 값을 가져올 수 있어요. 아래와 같이 활용할 수 있어보이네요 😊

```suggestion
  gap: ${({ theme }) => theme.spacing[3]};
```

Reply:
> 1. styled vs css + useTheme : 현재 요청한 코드는 styled 로 작성하였는데요, 이전 코드에서는 css + useTheme 으로 작성했거든요.. 혹시 어떤 것이 더 적합한 사용일지 궁금합니다.

질문 주신 것에서 정답은 없다고 생각해요! 다연님에게 편리한 방식으로 통일해서 작성해주시면 될 것으로 보여요. 한가지 차이점은 두 코드를 작성하면서도 느끼셨을 거 같은데, styled는 props에서 theme의 타입을 추론해 사용할 수 있어 css와 같이 타입을 따로 작성하지 않아도 된다는 점이 있겠네요 😊